src/Main.elm

```diff
@@ -235,6 +249,17 @@ newGameButton =
             [ text "Start new game" ]
         ]
 
+languageButton : Language -> Bool -> Html Msg
+languageButton lang isActive =
+    let classes = if isActive then
+                     "btn btn-secondary active"
+                  else
+                     "btn btn-secondary"
+    in
+    label [ class classes ]
```

Comment:
You have a convenient `classList` helper which may be easier to use here https://package.elm-lang.org/packages/elm/html/latest/Html-Attributes#classList